docs/cedarling/cedarling-quick-start.md

```diff
@@ -10,14 +10,185 @@ tags:
+
+## Policy Store Setup
+
+To begin using Cedarling, we need to set up a policy store. We will use [Agama Lab](https://cloud.gluu.org/agama-lab) for this. Please follow this video guide to set up your policy store:
+
+![agama-lab-policy-store](../assets/agama-lab-policy-store.mp4)
+
+For the trusted issuer token metadata, paste in the following content:
+```json
+{
+  "accessTokens": {
+    "trusted": true,
+    "entity_type_name": "Jans::Access_token",
+    "required_claims": [
+      "jti",
+      "iss",
+      "aud",
+      "sub",
+      "exp",
+      "nbf"
+    ],
+    "principal_mapping": [
+      "Jans::Workload"
+    ]
+  },
+  "idTokens": {
+    "trusted": true,
+    "entity_type_name": "Jans::id_token"
+  },
+  "userinfoTokens": {
+    "trusted": true,
+    "entity_type_name": "Jans::Userinfo_token",
+    "principal_mapping": [
+      "Jans::User"
+    ]
+  }
+}
+```
+
+After following the guide, the policy store URI will be copied to the clipboard. We will need this in the next step.
+
+## Tarp Setup
+
+1. Open Tarp on your browser.
+   ![image](../assets/tarp-blank.png)
+2. Click on `Add Client` and fill in the following details:
+
+   * Issuer: `https://account.gluu.org`
+   * Client Expiry Date: One day after your current date
+   * Scopes: `openid`,`profile`, and `role`
+3. Click `Register`. 
+4. Click on `Cedarling`, then `Add Configurations`
+5. Paste in the following configuration, replacing `<Policy Store URI>` with your policy store URL. 
+  ```json
+  {
+      "CEDARLING_APPLICATION_NAME": "My App",
+      "CEDARLING_POLICY_STORE_URI": "<Policy Store URI>",
+      "CEDARLING_POLICY_STORE_ID": "gICAgcHJpbmNpcGFsIGlz",
+      "CEDARLING_LOG_TYPE": "std_out",
+      "CEDARLING_LOG_LEVEL": "INFO",
+      "CEDARLING_LOG_TTL": null,
+      "CEDARLING_USER_AUTHZ": "enabled",
+      "CEDARLING_WORKLOAD_AUTHZ": "disabled",
+      "CEDARLING_PRINCIPAL_BOOLEAN_OPERATION": {
+          "or": [
+              {"===": [{"var": "Jans::Workload"}, "ALLOW"]},
+              {"===": [{"var": "Jans::User"}, "ALLOW"]}
+           ]
+      },
+      "CEDARLING_LOCAL_JWKS": null,
+      "CEDARLING_POLICY_STORE_LOCAL": null,
+      "CEDARLING_POLICY_STORE_LOCAL_FN": null,
+      "CEDARLING_JWT_SIG_VALIDATION": "disabled",
+      "CEDARLING_JWT_STATUS_VALIDATION": "disabled",
+      "CEDARLING_MAPPING_USER": "Jans::User",
+      "CEDARLING_MAPPING_WORKLOAD": "Jans::Workload",
+      "CEDARLING_JWT_SIGNATURE_ALGORITHMS_SUPPORTED": [
+          "HS256",
+          "RS256"
+      ],
+      "CEDARLING_ID_TOKEN_TRUST_MODE": "none",
+      "CEDARLING_LOCK": "disabled",
+      "CEDARLING_LOCK_SERVER_CONFIGURATION_URI": null,
+      "CEDARLING_LOCK_DYNAMIC_CONFIGURATION": "disabled",
+      "CEDARLING_LOCK_LISTEN_SSE": "disabled"
+  }
+  ```
+6. Click `Save`. Tarp will validate your bootstrap and initialize Cedarling.
+
+## Authentication Flow
+
+1. On the Authentication Flow screen, click on the lightning icon to trigger an authentication flow.
```

Comment:
I think you need another video here to show what success looks like.

Reply:
Pending https://github.com/JanssenProject/jans/pull/11137